### ISSUE TITLE
Trying to add mobilenet architecture

### ISSUE THREAD

**Issue by Shiro-LK** [JonasHarnau/udacity_mle_capstone#1]
Hello,

I am trying to use a mobilenet architecture instead of VGG but I am stuck because of some shape errors 
for example : `InvalidArgumentError (see above for traceback): Incompatible shapes: [1,18,28,36] vs. [1,19,29,36]
	 [[Node: gradients/sub_2_grad/BroadcastGradientArgs = BroadcastGradientArgs[T=DT_INT32, _class=["loc:@sub_2"], _device="/job:localhost/replica:0/task:0/cpu:0"](gradients/sub_2_grad/Shape, gradients/sub_2_grad/Shape_1)]]
`

Do you have an idea why I have this error ? (I am using PASCAL_VOC2007 train/test)

I have also a question regarding the loss function, do you know why using 4* num_achors for y true `y_true[:, 4 * num_anchors:, :, :]` ?

[VOC2007v2.txt](https://github.com/JonasHarnau/udacity_mle_capstone/files/1848954/VOC2007v2.txt)

[mobilenet.py.txt](https://github.com/JonasHarnau/udacity_mle_capstone/files/1848952/mobilenet.py.txt)
EDIT: I maybe found one answer to my question. Mobilenet use a stride instead of a pooling so to compute the reduction size of the image we cannot use the formula 2^5 = 32 but we have to do a loop using math.ceil
